atlasdb-client/src/main/java/com/palantir/atlasdb/util/MetricsManager.java

```diff
@@ -90,7 +90,9 @@ private synchronized void deregisterMetric(String fullyQualifiedMetricName) {
     }
 
     public void deregisterMetricsWithPrefix(Class clazz, String prefix) {
-        String fqnPrefix = MetricRegistry.name(clazz, prefix);
+        // isEmpty() check required because MetricRegistry.name elides missing components.
```

Comment:
'elides' - could you replace this with some plainer English? Either 'omits' or 'skips' would be fine.

Reply:
Yep, sure. I'm... not sure what I was thinking of at the time.